images/runtime/dotnetcore/generateDockerfiles.sh

```diff
@@ -21,8 +21,8 @@ declare -r DOCKERFILE_TEMPLATE="$DIR/template.Dockerfile"
 declare -r RUNTIME_BASE_IMAGE_TAG_PLACEHOLDER="%RUNTIME_BASE_IMAGE_TAG%"
```

Comment:
We should also update the base image tag used -- even though this image should be able to re-use the [existing .NET runtime tag](https://github.com/microsoft/Oryx/blob/main/build/constants.yaml#L69) since it was previously generated for both buster and bullseye, it's good practice to update tag with the latest changes whenever updating and releasing a new runtime image.

This will involve the following steps:
- Run the `Oryx-BaseImages-DotNetCore` pipeline against this branch
- Update the `constants.yaml` file in the spot I linked above for the `dot-net-core-runtime-base-tag` property with the new base runtime image tag
- Run the `build/generateConstants.yaml` script locally to update the variables within the repository that depend on this new base runtime image tag

Reply:
I have considered this step. But as I have not updated any platform version, I thought to skip updating the tag. But as you suggested I will update the tag as well. Thanks